test/quantization/test_quantize_fx.py

```diff
@@ -429,53 +411,32 @@ def forward(self, x):
 
     @skipIfNoFBGEMM
     def test_save_observer_state_dict(self):
-        class TwoLayerLinear(nn.Module):
-            def __init__(self):
-                super(TwoLayerLinear, self).__init__()
-                self.fc1 = nn.Linear(5, 5)
-                self.fc2 = nn.Linear(5, 5)
-
-            def forward(self, x):
-                x = self.fc1(x)
-                return self.fc2(x)
-
-        class Model(nn.Module):
-            def __init__(self):
-                super(Model, self).__init__()
-                self.subm = TwoLayerLinear()
-                self.fc = nn.Linear(5, 5)
-
-            def forward(self, x):
-                x = self.subm(x)
-                x = self.fc(x)
-                return x
-
-        model = Model().eval()
+        orig = LinearModelWithSubmodule().eval()
+        model = orig
         qconfig_dict = {'': torch.quantization.get_default_qconfig('fbgemm')}
 
         # symbolically trace
         model = symbolic_trace(model)
         model = prepare_static_fx(model, qconfig_dict)
-
         # run it through input
         x = torch.randn(5, 5)
         model(x)
 
         quant = convert_static_fx(model)
 
         # save state_dict of model
-        import io
+        obs_dict = torch.quantization.get_observer_state_dict(model)
```

Comment:
Currently we seem to be doing this only for activation observers. As we move to QAT, we will need similar functionality, but we will need to consider weight fake-quants too. Is there a plan to add weight observers to the state dict?